lib/resources/github.css

```diff
@@ -0,0 +1,99 @@
+/*
+
+github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```

Comment:
Please indicate somewhere in LICENSE/README where this and other new third-party libraries included come from (including URLs) and their licensing.

Reply:
Done